lte/gateway/c/core/oai/oai_mme/oai_mme.c

```diff
@@ -100,28 +101,28 @@ int main(int argc, char* argv[]) {
       TASK_MAX, THREAD_MAX, MESSAGES_ID_MAX, tasks_info, messages_info, NULL,
       NULL));
 
+  /*
```

Comment:
I'm not so sure of the implications of moving this before initializing other main dependencies, can you please validate this by running `make integ_test` or `make precommit` on magma_test VM?

Reply:
Previously I had only tested service start and looked at the logs. Now I ran the integration test `s1aptests/test_attach_detach.py` and it was successful.

Most other `integ_tests` were also successful. There were some unrelated failures like `OSError: [Errno 113] No route to host` in `s1aptests/test_attach_detach_rar_tcp_data.py`